src/http/common/throttler/throttler.guard.ts

```diff
@@ -4,7 +4,7 @@ import { FastifyRequest } from 'fastify';
 
 @Injectable()
 export class ThrottlerBehindProxyGuard extends ThrottlerGuard {
-  protected getTracker(request: FastifyRequest): string {
+  protected async getTracker(request: FastifyRequest) {
```

Comment:
this is not an asynchronous method

Reply:
it is required by updated throttle lirary